src/api/users.ts

```diff
@@ -32,3 +45,55 @@ router.delete("/users/history/:courseId", async (req, res) => {
   await sql`DELETE FROM courses_completed WHERE user_id = ${user.id} AND course_id = ${courseId}`;
   res.send();
 });
+
+router.get("/users/progress/:planId", async (req, res) => {
+  const user = await authUser(req);
+  const planId = req.params.planId;
+
+  await sql.begin(async (sql) => {
+    const planDetails = await oneOrNull<PlanDetails>(
+      sql`SELECT pr.total_units, pr.required_courses
```

Comment:
When testing this locally, I'm getting an error here saying `column pr.total_units does not exist`

Reply:
Yeah, I tested on a local db and did not want to miss with og db. It should be fine to add that column to the supabase db right? I just don't have the exact data for every majors total_units it tends to be around 180-200

Reply:
I will defer to @KaiiSnorlax's judgement here, she's the database expert

Reply:
Feel free to add it to the DB! Just try not to overwrite/remove anything already in there.

Reply:
Also edit the schema.sql file too, if possible!